src/java.desktop/share/classes/javax/swing/plaf/basic/BasicScrollBarUI.java

```diff
@@ -1628,15 +1674,15 @@ public void actionPerformed(ActionEvent e) {
                     }
                 }
             } while (parent != null);
-            if(useBlockIncrement)       {
+            if (useBlockIncrement)       {
                 scrollByBlock(direction);
                 // Stop scrolling if the thumb catches up with the mouse
-                if(scrollbar.getOrientation() == JScrollBar.VERTICAL)   {
-                    if(direction > 0)   {
-                        if(getThumbBounds().y + getThumbBounds().height
+                if (scrollbar.getOrientation() == JScrollBar.VERTICAL)   {
+                    if (direction > 0)   {
+                        if (getThumbBounds().y + getThumbBounds().height
                                 >= trackListener.currentMouseY)
                                     ((Timer)e.getSource()).stop();
-                    } else if(getThumbBounds().y <= trackListener.currentMouseY)        {
+                    } else if (getThumbBounds().y <= trackListener.currentMouseY)        {
```

Comment:
GUess you are not changing this block so no need for formatting cleanup in this PR.. infact the "{" are also way off if you had to change...there are other non-touched area like l1593...better to cleanup in separate cleanup..

Reply:
Makes sense. Was a remnant from when I tried implementing a solution here instead. Reverted the changes in this block.